tools/cl-entry/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const (
+	pr = `Must have a Pull Request already open.
+  Enter PR # => `
+
+	kind = `Choose type, one of
+    1. bug
+    2. improvement
+    3. security
+    4. breaking-change
+    5. note
+    6. deprecation
+  Enter Kind => `
+
+	note = `Write a note, for example
+	build: Added make target for creating changelog entries
+  Enter Note => `
+)
+
+var noteRe = regexp.MustCompile(`[a-z0-9/]+: .+`)
+
+func main() {
+	pr, err := ask(pr)
+	check(err)
+
+	n, err := ask(kind)
+	check(err)
+
+	label, err := label(n)
+	check(err)
+
+	body, err := askStr(note)
+	check(err)
+
+	msg, err := cleanup(body)
+	check(err)
+
+	file, err := write(pr, label, msg)
+	check(err)
+
+	fmt.Println("Created", file)
+}
+
+func write(pr int, label, msg string) (string, error) {
+	filename := filepath.Join(".changelog", fmt.Sprintf("%d.txt", pr))
+	sb := new(strings.Builder)
+	sb.WriteString("```release-note:")
+	sb.WriteString(label)
+	sb.WriteString("\n")
+	sb.WriteString(msg)
+	sb.WriteString("\n")
+	sb.WriteString("```")
```

Comment:
```suggestion
	sb.WriteString("```\n")
```

I don't remember if we have any rules for adding a line break at the end of the file, but my editor adds one by default, so this may be a "me" kind of problem, feel free to ignore it 😅